### PR TITLE
clear button behavior changed to fetch data onclick

### PR DESCRIPTION
**Pull request by vikchupak**

### CODE REVIEW

File: src/components/mainPage/CatalogSidebar.jsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ const CatalogSidebar = () => {
 
   const onClearFiltyers = () => {
```
Comment (vladyslav-dotsenko) [inline src/components/mainPage/CatalogSidebar.jsx:16]: Typo in word "filters". Please check project wide

**Review by vladyslav-dotsenko — CHANGES_REQUESTED** [review]

File: src/components/mainPage/MobileDrawer.jsx [hunk 2]
```diff
@@ -24,6 +24,8 @@ const MobileDrawer = ({ isOpen, onClose }) => {
 
   const onClearFiltyers = () => {
```
Comment (vikchupak) [inline src/components/mainPage/MobileDrawer.jsx]: just test comment

Comment (vikchupak) [inline src/components/mainPage/MobileDrawer.jsx]: another test comment

**Review by vikchupak — COMMENTED** [review]
review test submit

File: src/components/mainPage/CatalogSidebar.jsx (hunk 1 above)
Comment (vikchupak) [reply src/components/mainPage/CatalogSidebar.jsx:16]: Agree. Already fixed. Thanks.

**Review by vladyslav-dotsenko — APPROVED** [review]

**Pull request merged**